verilog/analysis/verilog_linter.h

```diff
@@ -34,19 +35,44 @@
 
 namespace verilog {
 
+// Base class of LintRuleStatus handlers
+class LintStatusHandler {
+ public:
+  LintStatusHandler() {}
+
```

Comment:
Make sure to add a virtual destructor. Not every compiler is happy without.

Reply:
Done